Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.h

```diff
@@ -46,8 +46,13 @@ class RemoteScrollbarsController final : public WebCore::ScrollbarsController {
     void mouseMovedInContentArea() final;
     void mouseEnteredScrollbar(WebCore::Scrollbar*) const final;
     void mouseExitedScrollbar(WebCore::Scrollbar*) const final;
+    bool shouldScrollbarParticipateInHitTesting(WebCore::Scrollbar*) final;
+    
+    void setScrollbarVisibilityState(ScrollbarOrientation, bool) final;
 
 private:
+    bool m_horizontalScrollbarIsVisible { false };
```

Comment:
We don't initialize these to the correct values for non-overlay scrollbars, so they aren't always correct, and there's no code to handle toggles between overlay and non-overlay scrollbars when the user changes the system scrollbar style.

Reply:
I suggest you either name these to make it explicit that they only apply to overlay scrollbars, or make sure the state is always correct.

Reply:
Right, renamed to specify this is only for overlay scrollbars.